primo/data_parser/well_data.py

```diff
@@ -272,19 +298,35 @@ def get_fully_partitioned_data(self):
 
         return None
 
-    def get_high_priority_wells(self, num_wells: int):
-        """Returns the top n wells by priority"""
+    def get_high_priority_wells(self, num_wells: int, sort: bool = True):
+        """
+        Returns the top n wells by priority
+
+        Parameters
+        ----------
+        num_wells : int
+            Returns the top `num_wells` number of wells
 
-        if not hasattr(self._col_names, "priority_score"):
+        sort : bool, default = True
```

Comment:
Would recommend changing the `sort` argument to something like `sort_by_priority` to make the intention super clear. `sort == True` could imply sorting in ascending order and it is not clear what index is used

Reply:
Sure, I will make the change.